apis/object/v1alpha2/types.go

```diff
@@ -131,6 +134,16 @@ type Readiness struct {
 	// +kubebuilder:validation:Enum=SuccessfulCreate;DeriveFromObject;AllTrue
```

Comment:
I think this Enum needs to be updated with the new value

Reply:
Oh heck - good catch. I'll update it and push here shortly